vault/logical_system_test.go

```diff
@@ -5665,13 +5666,6 @@ func TestSystemBackend_LoggersByName(t *testing.T) {
 			true,
 			false,
 		},
-		{
```

Comment:
Checking why this was removed

Reply:
The original PR added it. I think it's fine.